tests/server/TestConfig.js

```diff
@@ -59,15 +59,6 @@ for (const appName of appsList) {
       assert.equal(clientConfig.apiHost, 'https://addons.allizom.org');
       assert.equal(conf.util.getEnv('NODE_ENV'), 'stage');
     });
-
-    it(`should provide a development conf for ${appName}`, () => {
```

Comment:
I couldn't really see why this test (or the related ones) is useful for our suite because it's covering what the [node-config] test suite already covers. Did they exist just as a safety check against `node-config` regressions? Or to make sure our override configs exist?